frontend/src/scenes/batch_exports/batchExportEditLogic.ts

```diff
@@ -61,6 +61,8 @@ const formFields = (
                   schema: !config.schema ? 'This field is required' : '',
                   table_name: !config.table_name ? 'This field is required' : '',
                   has_self_signed_cert: false,
+                  exclude_events: '',
+                  include_evnts: '',
```

Comment:
`s/include_evnts/include_events` here and below

Reply:
Quick search and replace coming up! Thank you!